polaris.shopify.com/src/components/TokenList/TokenList.tsx

```diff
@@ -446,6 +446,9 @@ function TokenPreview({name, value}: TokenPreviewProps) {
         {...previewDivAttributes}
         style={{
           display: 'flex',
+          background: 'var(--p-color-bg-app)',
```

Comment:
The correct color to use here is `--p-color-bg` (`--p-color-bg-app` is deprecated)

```suggestion
          background: 'var(--p-color-bg)',
```